sources/packages/api/src/testHelpers/controllers/auth-test/auth-test.controller.ts

```diff
@@ -51,5 +58,34 @@ export class AuthTestController {
    */
```

Comment:
I think, the method does not have `* @param userToken`

Reply:
I will remove it!

Reply:
Removed!